controllers/userController.js

```diff
@@ -0,0 +1,62 @@
+const User = require('../models/userModel');
+const bcrypt = require('bcrypt');
+
+const userRegistration = (req, res) => {
+    const { name, userName, email, password } = req.body;
+    const user = new User(req.body);
+
+    user
+        .save()
+        .then(newUser => {
+            res.status(201).json(newUser);
+        })
+        .catch(err => {
+            res.status(500).json({ errMsg: "Could not create user account." });
+        })
+}
+
+// const getUsers = (req, res) => {
+
+//     User
+//         .find()
+//         .populate('notes')
+//         .then(users => {
+//             res.status(200).json(users);
+//         })
+//         .catch(err => {
+//             res.status(500).json({ errMsg: 'Could not retrieve users' });
+//         })
+
+// }
+
+const updateUser = (req, res) => {
+    const { id } = req.params;
+    const update = (req.body);
```

Comment:
For this. If you are trying to create and object with the properties from the body, it would be best to do it like

```const update = {...req.body};```

Unless that's not what you were trying to do lol